.cspell/custom-dict.txt

```diff
@@ -8,6 +8,7 @@ jetsons
 libc
 LIG
 mdbook
+mypluginsource
```

Comment:
MyPluginSource doesn't fit ?

Reply:
when the command run locally, this word is reported as an error, so I added it

Reply:
I bet you that if you use MyPluginSource instead of mypluginsource you won't have any error